packages/com.sncommunity.standard-blue-theme/src/main.css

```diff
@@ -0,0 +1,67 @@
+:root {
+
+  --sn-stylekit-editor-font-family: 'Lato', sans-serif;
+  --sn-stylekit-sans-serif-font: 'Lato', sans-serif;
+
+  --sn-stylekit-neutral-color: #6B97FF;
+  --sn-stylekit-neutral-contrast-color: white;
+
+  --sn-stylekit-info-color: #4279FF;
+  --sn-stylekit-info-contrast-color: white;
+
+  --sn-stylekit-success-color: #3965CA;
+  --sn-stylekit-success-contrast-color: white;
+
+  --sn-stylekit-warning-color: #f6a200;
+  --sn-stylekit-warning-contrast-color: white;
+
+  --sn-stylekit-danger-color: #f80324;
+  --sn-stylekit-danger-contrast-color: white;
+
+  --sn-stylekit-shadow-color: #d7e3fc;
+  --sn-stylekit-background-color: #edf2fb;
+  --sn-stylekit-border-color: #abc4ff;
+  --sn-stylekit-foreground-color: black;
+
+  --sn-stylekit-contrast-background-color: #e2eafc;
+  --sn-stylekit-contrast-foreground-color: #28385C;
+  --sn-stylekit-contrast-border-color: #abc4ff;
+
+  #blocks-editor hr:after {
+  background-color: #abc4ff;
+}
```

Comment:
This should be placed outside of the :root brackets.

Be sure to validate your css, you can try something like https://jigsaw.w3.org/css-validator/#validate_by_input or https://www.cssportal.com/css-validator/ if you are not comfortable with a cli or code editor.